subfun/BIDS_copyRawFolder.m

```diff
@@ -62,7 +62,7 @@ function BIDS_copyRawFolder(opt, deleteZippedNii)
         subNumber = group(iGroup).subNumber{iSub} ; % Get the subject ID
         
         % the folder containing the subjects data
-        subFolder = ['sub-', groupName, subNumber ];
+        subFolder = ['sub-', subNumber ];
```

Comment:
but if we have a groupe name before the subject number in a given BIDS data set we should keep that, no?